tests/test_amgx_direct_solver.cu

```diff
@@ -122,85 +122,81 @@ BOOST_AUTO_TEST_CASE(amgx_2_procs)
   cudaError_t cuda_error_code = cudaGetDeviceCount(&n_devices);
   mfmg::ASSERT_CUDA(cuda_error_code);
   int comm_size = dealii::Utilities::MPI::n_mpi_processes(MPI_COMM_WORLD);
-  if ((n_devices == 2) && (comm_size == 2))
+  if (comm_size == 2)
```

Comment:
Do we want that `if` at all then?

Reply:
Ping

Reply:
I don't see a reason why the test should not work with just one `MPI` process. Let's see what `CI` says in another pull request.

Reply:
See #206.